less/index.less

```diff
@@ -49,11 +51,134 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+// Variables
+
+@text-color: @firebrick;
+@firebrick: #B22222;
+@main-font: Georgia;
+
+
+// Mixins
+.bottom-content () {
+	
+	display: flex;
+	justify-content: center;
+
+}
+
+.intro-content () {
+	font-size: 2rem;
+}
+// General Styles
+
 * {
     box-sizing: border-box;
 }
 
+html {
+	
+	font-family: @main-font;
+  }
+
 html, body {
-    height: 100%;
+	height: 100%;
+	max-width: 100%;
+	@media (max-width: 500px) {
+		max-width: 100%;
+		background-color: pink;
+		display: flex;
+		flex-direction: column;
+		}
+
+}
+
+.intro-content img {
+	max-width: 100%;
+	height: auto;
 }
 
+.container {
+	
+	max-width: 100%;
+	height: 100%;
+	padding: 5% 5%;
+	margin: 0% auto;
+	}
+
+.top-nav {
+	width: 80%;
+	padding-bottom: .05%;
+	font-size: 2rem;
+	align-content:center;
+}
+.top-nav a {
+	display: flex;
+	color: @text-color;
+	padding: 0.125%;
+	padding-bottom: .07%;
+	padding-top: .07%;
+}
+.intro-content h1 {
+	font-size: 3rem;
+	display: flex;
+	padding-left: 37%;
+	max-width: 100%;
+	
+	font-style: italic;
+	padding-top: 7%;
+	padding-bottom: 1%;
+	
+}
+.intro-content img {
```

Comment:
I would encourage you to go back and revisit nesting styles with LESS. There are some docs in the slack notepad or even rewatch Josh's video.